packages/trip-form/package.json

```diff
@@ -11,7 +11,8 @@
   "dependencies": {
```

Comment:
Enable TypeScript checks for the new `TripOptions` portion of this package by adding a `tsconfig.json` and, in the script section of this file (and fix the errors shown when running `yarn`):
```
    "tsc": "tsc src/TripOptions/*.ts?"
```

Reply:
"resolved" in d9067256c61bba1caaebb9a456f6f58f0f278316, although I added the tsconfig.json that will allow tsc to run correctly by just running `tsc`